components/crypto/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "fips"
+name = "crypto"
```

Comment:
why rename

Reply:
The crates contains not only FIPS utilities, but also crypto operations (RNG). 
See doc comments in lib.rs